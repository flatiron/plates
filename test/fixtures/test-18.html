<span class="username"></span>
<span class="login name"></span>
