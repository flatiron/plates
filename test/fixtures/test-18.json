{
  "username": "lord thingy",
  "name": "Tom Marvolo Riddle"
}
