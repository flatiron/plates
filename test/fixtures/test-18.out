<span class="username">lord thingy</span>
<span class="login name">Tom Marvolo Riddle</span>
